fmf/base.py

```diff
@@ -544,3 +547,58 @@ def node(reference):
                 "No tree node found for '{0}' reference".format(reference))
         # FIXME Should be able to remove .cache if required
         return found_node
+
+    def get_raw_data_node(self):
```

Comment:
Could this be a private method? Personally I don't see why user of fmf library would call this.
However if there is need for it being public docstring will be very nice to have

Reply:
I agree, probably nobody just FMF hackers would use this method, so private is fine here.